spec/controllers/portal/portal_controller_spec.rb

```diff
@@ -33,24 +34,49 @@
     end
 
     context "as an authenticated client" do
-      let(:client) { create :client }
+      before { sign_in client }
 
-      before do
-        create :tax_return, year: 2018, client: client
-        create :tax_return, year: 2017, client: client
-        create :tax_return, year: 2020, client: client
-        sign_in client
-      end
+      context "with onboarding still to do" do
+        let(:client) { create :client, intake: (create :intake, current_step: "/en/questions/additional-info") }
 
-      it "is ok" do
-        get :home
+        before do
+          create :tax_return, year: 2018, status: :intake_in_progress, client: client
+          create :tax_return, year: 2017, status: :intake_in_progress, client: client
+        end
+
+        it "is ok" do
+          get :home
+
+          expect(response).to be_ok
+        end
 
-        expect(response).to be_ok
+        it "exposes current_step and tax_returns are empty" do
+          get :home
+
+          expect(assigns(:current_step)).to eq "/en/questions/additional-info"
+          expect(assigns(:tax_returns)).to eq []
+        end
       end
 
-      it "loads the client tax returns in desc order" do
-        get :home
-        expect(assigns(:tax_returns).map(&:year)).to eq [2020, 2018, 2017]
+      context "post-onboarding" do
+        let(:client) { create :client, intake: (create :intake) }
```

Comment:
would be good to see the current_step set in both cases, and ideally to show a client who has _just_ finished intake (is in ready for review)